package.json

```diff
@@ -35,5 +36,9 @@
     "node-pre-gyp": "amilajack/node-pre-gyp",
     "node-pre-gyp-github": "^1.4.3",
     "yarn": "^1.17.3"
+  },
+  "devDependencies": {
+    "cross-env": "^5.2.1",
+    "mocha": "^6.2.0"
```

Comment:
Any reason to go with mocha over jest? We use the latter in the browser and snapp (and we had issues w/ mocha tests previous for eg). 

Would be rad to keep it conistent (and migrating the current tests doesnt have to be too much of a pita: https://jestjs.io/docs/en/migration-guide )